.pre-commit-config.yaml

```
@@ -0,0 +1,15 @@
+repos:
+-   repo: https://github.com/charliermarsh/ruff-pre-commit
```

Давай еще такое добавим (прогони все файлы через него)
```suggestion
-   repo: https://github.com/pre-commit/pre-commit-hooks
    rev: v4.3.0
    hooks:
    -  id: trailing-whitespace
    -  id: end-of-file-fixer
-   repo: https://github.com/charliermarsh/ruff-pre-commit
```